packages/browser/src/stack-parsers.ts

```diff
@@ -55,14 +55,16 @@ export const chrome: StackLineParser = line => {
   return;
 };
 
+export const chromeStackParser: StackLineParser = [30, chrome];
```

Comment:
Can we move these numbers into constants? That way they can all be co-located at the top of the file, makes blame easier.